project/build.scala

```diff
@@ -0,0 +1,83 @@
+import sbt.Keys._
+import sbt._
+
```

Comment:
I asked this elsewhere but why the change to a Build.scala (also, note the `B`)? I didn't look closely but I saw no changes which warranted it